coordinator/src/orderbook/db/orders.rs

```diff
@@ -88,6 +93,7 @@ struct NewOrder {
 impl From<OrderbookNewOrder> for NewOrder {
     fn from(value: OrderbookNewOrder) -> Self {
         NewOrder {
+            id: Uuid::new_v4(),
```

Comment:
🔧 it's a bit unexpected that the transformation will generate a new id upon transforming. I would prefer that a transformation will always end up in the same result.

Reply:
The type `OrderbookNewOrder` does not have an id yet, only once the order is inserted into the DB using the local type `NewOrder` we will get the ID. This is what is happening here. 
Ideally this ID is generated by the db but I wasn't able to get this working.